apps/experiments/views/experiment.py

```diff
@@ -457,6 +469,43 @@ class DeleteFileFromExperiment(BaseDeleteFileView):
     pass
 
 
+# TODO: complete form
+class ExperimentVersionForm(forms.ModelForm):
+    class Meta:
+        model = Experiment
+        fields = ["description"]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["description"].widget.attrs.update({"class": "input input-bordered", "style": "width: 50%"})
+
+
+class CreateExperimentVersion(CreateView):
+    model = Experiment
+    form_class = ExperimentVersionForm
+    template_name = "experiments/create_version_form.html"
+    title = "Create Experiment Version"
+    button_title = "Create"
+    permission_required = "experiments.add_experiment"
+
+    # TODO: add logic
+    def form_valid(self, form):
+        form.instance.team = self.request.team
```

Comment:
Two things:
1. This view should also be a subclass of the `LoginRequiredMixin`
2. If you merge master into this branch (to bring the model changes) then we should be able to do the following:

```python
working_version = self.object
new_version = working_version.create_new_version()
# If we have a version specific description field, then we can pass in the description to the `create_new_version` method to create the version with that description
return HttpResponseRedirect(self.get_success_url())
```

Reply:
Merged https://github.com/dimagi/open-chat-studio/pull/642/commits/652155558a7c58a2e4e57fa786e55eb5cc2edfca